flatten-array.md

```diff
@@ -0,0 +1,19 @@
+The challenge is to write a function that accepts an arbitrarily-deep nested Array-like structure and returns a flattened structure with any null values removed without using the reduce method.
+
+For Example
+
+flattener.flatten([0, 2, [[2, 3], 8, 100, null, [[null]]], -2]);
```

Comment:
Here we would need to use some form of pseudo-code rather than something that can be taken for actual syntax.